src/real-time-vad.ts

```diff
@@ -53,19 +53,23 @@ export class MicVAD {
   audioNodeVAD: AudioNodeVAD
   listening = false
 
-  static async new(options: Partial<RealTimeVADOptions> = {}) {
+  static async new(
+    options: Partial<RealTimeVADOptions> = {},
+    deviceId?: string
```

Comment:
I'd prefer to move this optional deviceId parameter into the RealTimeVADOptions object above, something like 
```typescript
export interface RealTimeVADOptions
  extends FrameProcessorOptions,
    RealTimeVADCallbacks {
   deviceId?: string
}
```

Reply:
Just moved!

I initially wasn't sure if this approach made sense since `deviceId` was only relevant to the `MicVAD`, but it does feel cleaner this way.  Thanks for the suggestion.